06_timers/kernelspace_time/module_log.rst

```diff
@@ -0,0 +1,40 @@
+
+.. code-block:: bash
+
+	/modules # ./test.sh
+	[17575.101687] 'task_05' module initialized
+	cat /sys/class/time_class/absolute
+	This is a first reading
+	cat /sys/class/time_class/absolute
+	This is a first reading
+	iteration No1
+	delay = 3.454 sec
+	cat /sys/class/time_class/absolute
+	previous reading was at 4:52:58:651:332:303
+	cat /sys/class/time_class/absolute
+	previous reading was 3:513:46:542 ago
```

Comment:
What does it mean 4:52:58 and so on

Reply:
In the case of absolute reading, I use the following format: hours:minutes:seconds:milliseconds:microseconds:nanoseconds.

In case of relative reading: seconds:milliseconds:microseconds:nanoseconds.

P.S. I've just noticed a little mistake in my test. For relative reading, I print a wrong string BUT reading is performed from the desired file.